Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/ExtensionFunctions.kt

```diff
@@ -4,16 +4,15 @@ import android.content.Context
 import android.net.Uri
 import android.text.Html
 import android.text.Spanned
+import android.view.View
 import androidx.browser.customtabs.CustomTabsIntent
 import androidx.core.content.ContextCompat
 import com.mentalmachines.droidcon_boston.R
 
-
 fun String?.isNullorEmpty(): Boolean {
```

Comment:
isn't this method built-in?  not sure why you need this extension method

Reply:
I didn't add it, it was already here.